src/chat_statistics/stats.py

```diff
@@ -0,0 +1,73 @@
+from typing import Union
+from pathlib import Path
+import json
+from src.data import DATA_DIR
+
+import json
+from hazm import word_tokenize, Normalizer
+from wordcloud import WordCloud
+import arabic_reshaper
+from bidi.algorithm import get_display
+from loguru import logger
+
+
+class ChatStatistics:
+    """Generates chat statistics from a telegram chat json file
+    """
+    def __init__(self, chat_json: Union[str, Path]):
+        """
+        :param chat_json: path to telegram export json file
+        """    
+        # load chat data
+        logger.info(f'Loading chat data from {chat_json}')
+        with open(chat_json) as f:
+            self.chat_data = json.load(f)
+
+        self.normalizer = Normalizer()
+
+        # load stopwords
+        logger.info(f'Loading stopwords fomr {DATA_DIR / "stopwords.txt"}')
+        stop_words = open(DATA_DIR / "stopwords.txt").readlines()
+        stop_words = list(map(str.strip, stop_words))
+        self.stop_words = list(map(self.normalizer.normalize, stop_words))
+
+    def generate_word_cloud(
+            self,
+            output_di: Union[str, Path],
+            width: int = 500, height: int = 600,
+            max_font_size: int = 250
```

Comment:
Better to add other optional wordcloud arguments such as `background_color`.
```suggestion
    def generate_word_cloud(
            self,
            output_di: Union[str, Path],
            width: int = 500, height: int = 600,
            max_font_size: int = 250,
            background_color="white",
```